py_gtfs_rt_ingestion/lib/converter.py

```diff
@@ -69,6 +75,7 @@ def is_gtfs_rt(self) -> bool:
             self.BUS_TRIP_UPDATES,
             self.BUS_VEHICLE_POSITIONS,
             self.VEHICLE_COUNT,
+            self.LIGHT_RAIL,
```

Comment:
This might not be GTFS data, a comment or something about this not being an implemented ConfigType and needing further review may be helpful.